config/test_system_monitor.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I know it is stupid, but can you please name the config as other configs? (too many "conventions" it is deadly)
```
(osgar) md@md-ThinkPad-P50:~/git/osgar$ python -m osgar.record config/test
test-canserial.json            test-lord-imu.json             test-replay-node.json          test-vesc.json
test-fusion.json               test-lord-imu-linux.json       test-robik.json                test-wifiscan.json
test-gas-detector.json         test-opencv-camera.json        test-spider-gps-imu.json       test-windows-gps.json
test-gps-imu.json              test-pcan.json                 test-spider.json               test-winsen-gas-detector.json
test-ip-camera.json            test-pulutof1.json             test_system_monitor.json       test-zeromq.json
test-lidar-tim571.json         test-realsense-D400.json       test-usb-camera.json           
test-lora.json                 test-realsense-L500.json       test-usb-lidar.json            
test-lora-windows.json         test-realsense-T200.json       test-velodyne.json             
```

Reply:
Ok